src/pykeen/datasets/cli.py

```diff
@@ -261,25 +277,24 @@ def expected_metrics(dataset: str, max_triples: Optional[int], log_level: str):
             ks = (1, 3, 5, 10) + tuple(
                 10**i for i in range(2, int(math.ceil(math.log(dataset_instance.num_entities))))
             )
-            this_metrics: MutableMapping[str, Mapping[str, float]] = dict()
-            for label, sides in dict(
-                head=[LABEL_HEAD],
-                tail=[LABEL_TAIL],
-                both=[LABEL_HEAD, LABEL_TAIL],
-            ).items():
-                num_candidates = df[[f"{side}_candidates" for side in sides]]
+            metrics = [
+                ArithmeticMeanRank(),
+                InverseArithmeticMeanRank(),  # needs simulation
+                HarmonicMeanRank(),  # needs simulation
+                InverseHarmonicMeanRank(),
+                GeometricMeanRank(),  # needs simulation
+                InverseGeometricMeanRank(),  # needs simulation
+                *(HitsAtK(k) for k in ks),
+            ]
+            this_metrics: MutableMapping[ExtendedTarget, Mapping[str, float]] = dict()
+            for label, sides in SIDE_MAPPING.items():
+                num_candidates = df[[f"{side}_candidates" for side in sides]].values.ravel()
```

Comment:
@mberr is this right? for the ones with two columns, this just combines them all together

Reply:
Yes, sounds right.